<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_Triangle</class>
 <widget class="QWidget" name="Form_Triangle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>21</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>X1:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>X2:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>X3:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>21</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Y1:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Y2:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Y3:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>62</y>
     <width>71</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_3"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>71</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLineEdit" name="lineEdit_4"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_5"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_6"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>260</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Create!</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
